src/main/webapp/resources/scripts/controllers/MapController.coffee

```diff
@@ -1,9 +1,29 @@
-angular.module('calendar').controller 'MapController', ($rootScope, $scope) ->
-  new class
-    constructor: ->
-      $rootScope.title = "Map"
-      $scope.mapOptions = {
-        center: new google.maps.LatLng(50.061389, 19.938333),
-        zoom: 15,
-        mapTypeId: google.maps.MapTypeId.ROADMAP
-      }
+angular.module('calendar').controller 'MapController', ($rootScope, $scope, Event) ->
+  $rootScope.title = "Map"
+
+  $scope.markers = []
+
+  hidePointer = ->
+    $scope.pointer.setMap(null)
```

Comment:
pointer here is undefined when saving/cancelling event with initialized map

(add or edit event with map in background)